pubsub/google/cloud/pubsub_v1/publisher/client.py

```diff
@@ -133,7 +134,9 @@ def __init__(self, batch_settings=(), **kwargs):
         # The batches on the publisher client are responsible for holding
         # messages. One batch exists for each topic.
         self._batch_lock = self._batch_class.make_lock()
+        self._stoping_lock = threading.Lock()
```

Comment:
nit: spelling - "stopping_lock"

Reply:
Wow, yes! I was in a hurry, terribly sorry